cl/scrapers/tasks.py

```diff
@@ -650,3 +660,198 @@ def update_docket_info_iquery(self, d_pk: int, court_id: str) -> None:
         tag_names=None,
         skip_iquery_sweep=True,
     )
+
+
+def process_scotus_captcha_transcription(transcription: str) -> str:
+    """Converts the transcription of a SCOTUS audio CAPTCHA into a list of five alphanumeric characters. Throws an error if it is unable to do so.
+
+    SCOTUS audio CAPTCHAs consist of three alphanumeric characters read out using the NATO phonetic alphabet. This makes extracting them from a transcription quite easy. The main challenge is that certain transcription models may choose odd spellings of certain phonetic characters, alternatively spell or not spell numbers, and insert punctuation in arbitrary locations. This method attempts to deal with all of those.
+
+    :param transcription: The transcription of the CAPTCHA.
+    :return: The alphanumeric characters extracted from the transcription."""
+
+    numeric_map = {
+        "zero": "0",
+        "one": "1",
+        "two": "2",
+        "three": "3",
+        "four": "4",
+        "five": "5",
+        "six": "6",
+        "seven": "7",
+        "eight": "8",
+        "nine": "9",
+    }
+
+    words = [
+        re.sub(r"\W+", "", word) for word in transcription.lower().split(" ")
+    ]
+
+    if len(words) != 5:
```

Comment:
Running some tests I found events that raised:

```
File "/opt/courtlistener/cl/scrapers/tasks.py", line 786, in subscribe_to_scotus_updates
    solution = process_scotus_captcha_transcription(transcription)
  File "/opt/courtlistener/cl/scrapers/tasks.py", line 691, in process_scotus_captcha_transcription
    raise ValueError(f"Expected 5 words, got {len(words)}")
ValueError: Expected 5 words, got 4
```

Can we confirm that all Kendo CAPTCHAs contain exactly five words?

If so, I think this is correct, since the solution seems wrong. In that case, the task will be retried, a new CAPTCHA will be requested, and hopefully it will be solved correctly.